html/webappapis/scripting/event-loops/microtask_after_raf.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<head>
+<title>Microtasks: clean up after running script </title>
```

Comment:
Nit: trailing whitespace in the title, and title doesn't seem to match the filename (and is equal to the title of the second test file)